src/components/AppHeader/AppHeader.tsx

```diff
@@ -34,9 +36,14 @@ const AppHeader = () => {
     const [isNotificationServiceEnabled] = useGrowthbookGetFeatureValue({
         featureFlag: 'new_notifications_service_enabled',
     });
+    const { isCheckingOidcTokens } = useIsLoadingOidcStore(
+        useShallow(state => ({
+            isCheckingOidcTokens: state.isCheckingOidcTokens,
+        }))
+    );
 
     const renderAccountSection = () => {
-        if (!isEndpointPage && !activeAccount) {
+        if ((!isEndpointPage && !activeAccount) || isCheckingOidcTokens) {
```

Comment:
i added a new store to keep track of the loader for when we check between OIDC tokens and auth list tokens, and needed it in a store because we should show a loader for both the header and the main content